test/services/lduService.test.js

```diff
@@ -0,0 +1,53 @@
+const createLduService = require('../../server/services/lduService')
+
+const deliusClient = { getAllLdusForProbationArea: jest.fn() }
+const lduActiveClient = { allActiveLdusInArea: jest.fn(), updateWithActiveLdu: jest.fn() }
+const onlyActiveLdus = ['ABC123', 'DEF345']
+const probationAreaCode = 'N02'
+const activeLdus = {}
+
+describe('lduService', () => {
+  afterEach(() => {
+    jest.restoreAllMocks()
+  })
+  describe('getActiveLdusForProbationArea', () => {
+    it('Should return an array of only the active LDUs', async () => {
+      lduActiveClient.allActiveLdusInArea.mockResolvedValue(onlyActiveLdus)
+      const result = await createLduService(
+        deliusClient.getAllLdusForProbationArea,
```

Comment:
not sure this is right? shouldn't the client be passed through rather than just one of its methods?

Reply:
hmm. I followed what I thought the same pattern as other similar tests.